core/modules/notifications/client/notifications.js

```diff
@@ -81,6 +83,14 @@ Template.notification.events({
   },
 });
 
+Template.notifications.events({
+  'click .js-notification-mark-all-as-read'(event) {
+    event.preventDefault();
+    markAllNotificationsAsRead();
+    closeModal('notifications');
```

Comment:
usually not a good UX practice to launch two actions on a single click

Reply:
fixed in https://github.com/Refty/lemverse/pull/26/commits/fe2c8ac15fd9cde9765d77b3cab9f79ad8d9670d